### PR TITLE
Docs update platform operator

### PR DESCRIPTION
**Pull request by PolinaSadykova**
changes: Operator to Platform Operator and Partner vs Operator roles

### CODE REVIEW

File: CONTRIBUTING.md
Comment (Jelledb) [inline CONTRIBUTING.md:15]:
```suggestion
* **Partners** have an administrator role. They are responsible for cloud management in the organization by providing meshStack to their internal *customers* / *users*. Partners can access customers' resources to support the deployment of resources or to perform other administrative tasks such as cost control, cloud compliance, permission management, etc (some of these roles might be taken by different people than Partners, however, we include them to the combined Partner role, as it represents an administrative role)..There is usually a contractual agreement between *partners* and *customers* that defines the obligations and rights of the *partner*.
```

File: CONTRIBUTING.md
Comment (Jelledb) [inline CONTRIBUTING.md:16]:
```suggestion
* **Platform Operators** are also an administrative role but with specific technical backgrounds and responsibilities for operating and providing cloud technology to *users*, which include monitoring of cloud operations, integrating platforms, providing cloud services, and ensuring cloud compliance.
```

File: CONTRIBUTING.md
Comment (Jelledb) [inline CONTRIBUTING.md:17]:
```suggestion
All documentation is split between the User Docs and Administrator Docs. User Docs are more oriented toward the basics of meshStack and could be valuable for all roles. Admistrator Docs are solely addressed to Partners and Platform Operators.
```

File: docs/administration.index.md
Comment (Jelledb) [inline docs/administration.index.md:7]:
```suggestion
Assigning Partner Users can only be done by Partner Admins.
```

I just checked this in meshStack. Only Partner Admins can do this.

File: docs/meshstack.aws.landing-zones.md
Comment (Jelledb) [inline docs/meshstack.aws.landing-zones.md:32]:
```suggestion
Platform Operators can also configure an [CloudFormation StackSet](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/what-is-cfnstacksets.html) on an AWS Landing Zone. meshStack will then ensure that all AWS Accounts provisioned using this Landing Zone will receive a StackInstance from the StackSet. This allows Platform Operators to leverage that StackSet to centrally manage configuration and resources for all AWS Accounts under the landing zone.
```

File: docs/meshstack.billing-configuration.md
Comment (Jelledb) [inline docs/meshstack.billing-configuration.md:26]:
```suggestion
- **Private Cloud billing**: Partners can define products and usage rates in any currency.
```

File: docs/meshstack.billing-configuration.md
Comment (Jelledb) [inline docs/meshstack.billing-configuration.md:31]:
```suggestion
- **Tenant Fees**: Partners can define tenant fees in any currency.
```

File: docs/meshstack.billing-configuration.md
Comment (Jelledb) [inline docs/meshstack.billing-configuration.md:110]:
```suggestion
When configuring a product for private cloud billing, Partners need to choose the cloud resource type to target and how the metering engine
```

File: docs/meshstack.billing-configuration.md
Comment (Jelledb) [inline docs/meshstack.billing-configuration.md:112]:
```suggestion
Traits are a properties of a cloud resource like CPU or RAM. Partners can use traits to configure predicates (filter resources applicable to a pricing rule) and to control how meshMetering calculates usages.
```

File: docs/meshstack.cloudfoundry.metering.md
Comment (Jelledb) [inline docs/meshstack.cloudfoundry.metering.md:15]:
```suggestion
Resources with the following traits are addressable in the [Product Catalog](meshstack.billing-configuration.md#defining-a-custom-product-catalog). Platform Operators can use the traits of these resources to define fine-granular product and pricing rules.
```

File: docs/meshstack.gcp.landing-zones.md
Comment (Jelledb) [inline docs/meshstack.gcp.landing-zones.md:15]:
```suggestion
Folders and the application of organization constrains on the projects contained in them through the use of [Organization Policy Service](https://cloud.google.com/resource-manager/docs/organization-policy/overview) can be setup outside of meshcloud by a Platform Operator.
```

File: docs/meshstack.identity-provider.md
Comment (Jelledb) [inline docs/meshstack.identity-provider.md:6]:
```suggestion
Partners must be aware of the following requirements and limitations for IdPs serving as identity sources for meshStack.
```

File: docs/meshstack.identity-provider.md
Comment (Jelledb) [inline docs/meshstack.identity-provider.md:17]:
```suggestion
meshStack supports the simultaneous integration of multiple [Enterprise Identity Providers (IdPs)](https://en.wikipedia.org/wiki/Identity_provider) at the Identity Broker. This allows Partners to combine identities from different sources in the platform and unify management of multi-cloud access in meshStack.
```

File: docs/meshstack.index.md
Comment (Jelledb) [inline docs/meshstack.index.md:52]:
```suggestion
meshcloud will typically operate your meshStack installation as a [managed service](./meshstack.managed-service.md) for you. As a managed service, all configuration and validation is done by meshcloud. Nonetheless, we make references to configuration options in the documentation so that Partners get a better understanding of meshStack's capabilities. The configuration references also simplify examples and communicate the exact parameters that may need to be supplied by Partners.
```

File: docs/meshstack.logging.md
Comment (Jelledb) [inline docs/meshstack.logging.md:71]:
```suggestion
| Changes to logging configuration (especially deactivation of logging) | Can only be done by meshcloud, no logging atm |
```

File: docs/meshstack.meshmarketplace.metering.md
Comment (Jelledb) [inline docs/meshstack.meshmarketplace.metering.md:136]:
```suggestion
Partners can mark sellers as "out of scope" via configuration. Once a seller is marked as out of scope,
```

File: docs/meshstack.tenants.md
Comment (Jelledb) [inline docs/meshstack.tenants.md:10]:
```suggestion
meshStack does not support reusing platformTenants. Imagine a meshProject has been set as [deleted by a platform operator](administration.projects.md#delete-projects), but the underlying platform tenant has not been deleted by the platform operator. You may think of reusing this tenant in another project by using the [meshObject Import](https://docs.meshcloud.io/api/#mesh_object_declarative_import). This is not supported by meshStack and will lead to an error during import. The reason for this current limitation is handling such a reuse of a tenant in meshMetering. Having the same tenant being used by different projects (perhaps even within the same month), makes it hard or in some cases even impossible to decide which project to charge for this tenant. Therefore reusing platformTenants is currently not supported by meshStack.
```

**Review by Jelledb — APPROVED** [review]

**Comment by JohannesRudolph** [comment]
Why does this PR have 152 commits 😨  maybe a good idea to do some git magic to rewrite/squash it before merging to develop

**Comment by henryde** [comment]
Don't merge the PR in this state, the commits are a total mess!

**Comment by henryde** [comment]
I've cleaned up the Git history, please use this PR instead: https://github.com/meshcloud/meshcloud-docs/pull/722

**Pull request closed without merging**
